index.html

```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
 
-    <div id='startStopButton'>Start</div>
+    <div id='startStopButton' class='enabled'>Start</div>
```

Comment:
This introduces a bug: the idea is that the button is disabled until WebRTC support is detected. Thus, if the browser doesn't support WebRTC it will remain disabled.